snap/snapcraft.yaml

```
@@ -145,23 +145,47 @@ parts:
       - libwayland-cursor0
       - libwayland-egl1
       - libwayland-server0
-      - libnvidia-egl-wayland1
     prime:
       - usr/lib
       - usr/share/doc/*/copyright
-      - usr/share/egl/egl_external_platform.d
 
-  gbm:
-    # GBM EGL external platform
-    plugin: nil
+  nvidia-egl-ext-deps:
+    # eglexternalplatform-dev >= 1.2
+    source: https://github.com/NVIDIA/eglexternalplatform/archive/refs/tags/1.2.tar.gz
```

This more git-y, I suppose?

```suggestion
    source: https://github.com/NVIDIA/eglexternalplatform.git
    source-tag: 1.2
    source-depth: 1
```

Will also yield better to SBOM generation with time.

---

Done